src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/postponementrequest/PostponementRequestAboutToStartHandlerTest.java

```diff
@@ -79,6 +81,19 @@ static Hearing getHearing(int hearingId) {
                 .build()).build();
     }
 
+    public static HearingOptions getHearingOptions() {
+        HearingOptions hearingOptions = HearingOptions.builder()
+                .wantsToAttend(YES)
+                .languageInterpreter(YES)
+                .languages("Spanish")
+                .signLanguageType("A sign language")
+                .arrangements(Arrays.asList("hearingLoop", "signLanguageInterpreter"))
+                .other("Yes, this...")
+                .build();
+
+        return  hearingOptions;
```

Comment:
Just add the return tot he actual builder no point in this if nothing else is happening.